src/Breadcrumbs/Breadcrumb.php

```diff
@@ -117,8 +117,16 @@ class Breadcrumb
      */
     public function __construct($name, $type, array $metaData = [])
     {
-        if (!is_string($name) || $name === '') {
-            throw new InvalidArgumentException('The breadcrumb name must be a non-empty string.');
+        if (!is_string($name)) {
+            if (is_null($name)) {
+                $name = 'NULL breadcrumb name given';
```

Comment:
All of these cases can be condensed to the same name, "`<no name>`" with a validation error in the breadcrumb metadata to indicate a configuration issue. That way, the presentation is consistent and any future interface to searching/filtering breadcrumbs can aggregate on a single name value or breadcrumbs with a BreadcrumbError, etc.

* Breadcrumb name is null?
    * `<no name>` - `['BreadcrumbError' => 'NULL provided as the breadcrumb name']`
* Breadcrumb name is empty?
    * `<no name>` - `['BreadcrumbError' => 'Empty string provided as the breadcrumb name']`
* Breadcrumb name is not a string?
    * `<no name>` - `['BreadcrumbError' => 'Breadcrumb name must be a string - "Integer" provided instead']`